var/spack/repos/builtin/packages/py-neuron-reduce/package.py

```diff
@@ -11,9 +11,10 @@ class PyNeuronReduce(PythonPackage):
 
     homepage = "https://github.com/orena1/neuron_reduce"
     url = "https://pypi.io/packages/source/n/neuron_reduce/neuron_reduce-0.0.6.tar.gz"
-    git = "git@github.com:orena1/neuron_reduce.git"
+    git = "git@github.com:BlueBrain/neuron_reduce.git"
```

Comment:
repo is not private anymore 
```suggestion
    git = "https://github.com/BlueBrain/neuron_reduce.git"
```